osvr/RenderKit/RenderManagerD3DBase.cpp

```diff
@@ -703,172 +703,6 @@ namespace renderkit {
       return true;
     }
 
-    bool RenderManagerD3D11Base::ComputeAsynchronousTimeWarps(
```

Comment:
Correct.  The base-class implementation was generalized to handle both cases and the duplicate slightly-different D3D implementation removed.